rskj-core/src/main/java/co/rsk/config/BridgeConstants.java

```diff
@@ -60,6 +60,9 @@
 
     protected int lockingCapIncrementsMultiplier;
 
+    protected int irisActivationBtcHeight;
```

Comment:
```suggestion
    protected int btcHeightWhenBlockIndexActivates;
```
Maybe a name that doesn't necessary ties to Iris is better, what do you think?